java/spacewalk-java.changes

```diff
@@ -1,3 +1,5 @@
+- Fix symlinks for gson, httcomponents on Leap 15.2
+
```

Comment:
So if I understand correctly, the symlinks will not change, but in the code you are accessing them via the new path now. Right?

Reply:
So, the build will use the `gson` and `httpcomponets` jars from Leap instead of the same jars from Uyuni (which have been removed anyways). Previously the build used the `gson` and `httpcomponents` packages from Uyuni but at install time the packages from Leap were installed.
Tomcat and Taskomatic load the jars referenced via the symlinks in `/srv/tomcat/webapps/rhn/WEB-INF/lib` and `/usr/share/spacewalk/taskomatic`. Since the symlinks for these jars were broken both Tomcat and Taskomatic failed to start.